adafruit_imageload/__init__.py

```diff
@@ -39,7 +39,11 @@ def load(filename, *, bitmap=None, palette=None):
             # meh, we tried
             pass
 
-    with open(filename, "rb") as file:
+    file_or_filename = filename
+    if isinstance(file_or_filename, str):
+        file_or_filename = open(filename, "rb")
+
+    with file_or_filename as file:
```

Comment:
I think the naming here is a bit confusing, because we always have a file in ``file_or_filename`` after those lines.
The clean version would be to rename the argument itself to ``file_or_filename``, and then do ``open_file = open(...)`` and use ``open_file`` everywhere beyond here.